schedule/autoyast_mini_staging.yaml

```diff
@@ -0,0 +1,20 @@
+name:           autoyast_mini_staging@64bit-staging
+description:    >
+    Test verifies installation with minimal autoyast profile. Same as autoyast_mini but with product defined using url and not as registered modules. 
+    Maintainer: riafarov@suse.com
```

Comment:
I assume this should mention your name, right?

Reply:
ehm...no idea. But Rodion came up with the test. So let's keep the credit in his pocket. :)